cloud-chatops/src/main.py

```diff
@@ -60,6 +60,9 @@ def run_pr(channel, mention=False) -> None:
     schedule.every().monday.at("09:00").do(
```

Comment:
something for a future PR - but maybe look into setting this via a yaml file or conf file 
Or better yet have a chat with me about setting up a stackstorm sensor to do this stuff!